hooks/gen_docs.py

```diff
@@ -0,0 +1,22 @@
+"""Generates the whole 'generatable' KPOps documentation"""
+import subprocess
+
+from hooks import PATH_ROOT
+
+PATH_KPOPS_MAIN = PATH_ROOT / "kpops/cli/main.py"
+PATH_CLI_COMMANDS_DOC = PATH_ROOT / "docs/docs/user/references/cli-commands.md"
+
+# Run typer-cli on kpops to generate doc on CLI usage
+# TODO: try to use typer_cli.main.docs here instead
+typer_args = (
+    f"typer {PATH_KPOPS_MAIN} utils docs --name kpops --output {PATH_CLI_COMMANDS_DOC}"
+)
+subprocess.run(typer_args, shell=True)
+
+# Replace wrong title in CLI Usage doc
+text: list[str] = []
```

Comment:
is it necessary to instantiate an empty list here before actually reading the file into it?

Reply:
No, I had assumed (wrongly) that the context manager creates a scope.